usaspending_api/awards/models.py

```diff
@@ -304,7 +324,8 @@ class Procurement(AwardAction):
     piid = models.CharField(max_length=50, blank=True)
     parent_award_id = models.CharField(max_length=50, blank=True, null=True, verbose_name="Parent Award ID")
     cost_or_pricing_data = models.CharField(max_length=1, blank=True, null=True)
-    type_of_contract_pricing = models.CharField(max_length=2, blank=True, null=True, verbose_name="Type of Contract Pricing")
+    type_of_contract_pricing = models.CharField(max_length=2, blank=True, null=True, choices=CONTRACT_PRICING_TYPES, verbose_name="Type of Contract Pricing")
```

Comment:
Is contract pricing type always a relevant piece of data for contracts? I'm trying to figure out if this pattern should match what we're doing on the award type/desc fields in the award model...where we default to `U`'/`Unknown`. But yeah, if contract pricing isn't always relevant, it makes sense to leave these as `NULL` with no default values.